.changeset/great-penguins-compete.md

```diff
@@ -0,0 +1,6 @@
+---
+"@internal/fuel-core": patch
+"@fuel-ts/abi-coder": patch
```

Comment:
```suggestion
"@fuel-ts/abi-coder": minor
```

Should this be a breaking change now we are throwing for this condition?

Reply:
I'd say it's not a breaking change because the API is the same.

But it's worth the reflection, and more context would be helpful.